styleguide/derek/pattern-components/challenges-river/_challenges-river.scss

```diff
@@ -0,0 +1,87 @@
+.challenges-wrapper {
+  margin-top: 50px;
+}
+
+.challenges-header,
+.challenges-subhead {
+  margin: 0;
+  margin-bottom: 25px;
+  padding-left: 25%;
+}
+
+.challenges-itemWrapper {
+  @include make-xs-column(6);
+
+  padding: 0;
+
+  :nth-child(odd) {
+    .challenges-item {
+      float: right;
+    }
+  }
+}
+
+.challenges-item {
+  @include make-xs-column(10);
+  @include make-md-column(8);
+
+  margin-bottom: 50px;
+
+  .rackshape {
+    height: auto;
+    width: 100%;
+  }
+
+}
+
+.challenges-icon {
+  @include make-xs-column(3);
+  @include make-lg-column(2.5);
+
+  padding: 0;
+}
+
+.challenges-description {
+  @include make-xs-column(9);
+  @include make-lg-column(8);
+
+}
+
+.challenges-descriptionTitle {
+  margin-top: 0;
+}
+
+@media screen and (max-width: $screen-xs-max) {
+  .challenges-description {
+    padding-right: 0;
+  }
+}
+
+@media only screen and (max-width: 500px) {
```

Comment:
why are we using a non-standard breakpoint here?

Reply:
@WidgetsBurritos This breakpoint was done previously for the platforms river( https://github.com/rackerlabs/zoolander/blob/master/styleguide/derek/pattern-components/platforms/_platforms.scss#L112 ). Since the challenges section is very similar to/a replacement for platforms, I replicated it for this pull request. At 767px it looked odd to me to have the wider "challenges".

I personally think it looks better with the custom breakpoint, but I'm not married to it. If design/dev feels its better to not have it, thats fine. 

I've gone ahead and made a new branch that can be used for comparison.
http://mte-3367-challenge-river-wide-xs.styleguide.dev.website.rackspace.com/derek/incubation/homepage

Reply:
Sure thing. Not sure if you want to discuss with @runrog and Mike. But let me know. Like I said, not married either way.